.github/l10n/linter_config.yml

```diff
@@ -27,6 +27,4 @@ CO01:
     - "{ -brand-name-mozilla } account"
   exclusions:
```

Comment:
Hmh I'm actually thinking about adding "MDN" as another brand to check, seems all is passing at the moment: [janbrasna@`c16e494`✅](https://github.com/janbrasna/bedrock/actions/runs/10525113769/job/29163220208#step:5:1)

Reply:
Nah, an "MDN" literal string sneaked in with [`3e34323`](https://github.com/mozilla/bedrock/commit/3e3432366069a3144dee988e9c40c18a19dddea9#diff-943e11c26372aa775b128c9ccabe81d5774d69c8d764c160db37bfcfe9cf9059R9) and already hit pontoon so it's too late for adding that here.